target/linux/ramips/dts/MAESTRO-E220.dts

```diff
@@ -0,0 +1,112 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "maestro-e220", "ralink,mt7620a-soc";
+	model = "Maestro E220";
```

Comment:
Full manufacturer name seems to be "Maestro Wireless Solutions", not just "Maestro".